source/assets/js/srf-carousels.js

```diff
@@ -128,48 +115,51 @@ export function init() {
             $carousel.slick('slickSetOption', 'slidesToScroll', slidesToShow, false);
         }
 
-        // unhide the following slidesToShow - 1 from screenreaders as well:
-        let currentPage = Math.floor(currentElement / slidesPerScreen) + 1;
-
         // remove not currently visible slides from tabindex
-        let to = (slidesPerScreen * currentPage) - 1; // zero indexed
-        let from = to - (slidesPerScreen - 1);
+        changeTabIndexIfNotVisible($carousel, slidesPerScreen, currentElement);
 
-        $carousel.find('.carousel__item').each(function (i) {
-            i >= from && i <= to
-                ? $(this).attr('aria-hidden', false).find('.article-video__link').attr('tabindex', 0)
-                : $(this).attr('aria-hidden', true).find('.article-video__link').attr('tabindex', -1);
-        });
-
-    }).on('afterChange', function (slick, currentSlide) { // instead of: ... .on('keyup', '.carousel__link--next', function (e) {
+    }).on('afterChange', function (slick, currentSlide) {
         // as soon as slick's ready, we put the focus on the current elm
         if (slidesPerScreen > 1) {
-            $(this).find('.slick-current .article-video__link').focus();
+            $(this).find('.slick-current a').focus();
```

Comment:
theoretisch: falls es mehrere `<a>`-tags in dem slide hätte, gäbs da ein problem mit dem focus

Reply:
Korrekt! Fixe das mit einem >